modules/dynamic-plugins/proc-export-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,124 @@
+----
+
+<1> `@backstage/plugin-notifications` package is treated as a private dependency and is bundled in the dynamic plugin package, despite being in the `@backstage` scope.
+<2> `@backstage/plugin-notifications-backend` package is marked as an embedded dependency and is bundled in the dynamic plugin package.
+--
+
+Front-end plugins::
++
+--
+Front-end plugins can use `scalprum` for configuration, which the CLI can generate automatically during the export process. The generated default configuration is logged when running the following command:
+
+.Example command to log the default configuration
+[source,bash]
+----
+npx @janus-idp/cli@latest export-dynamic
+----
+
+The following is an example of default `scalprum` configuration:
+
+.Default `scalprum` configuration
+[source,json]
+----
+"scalprum": {
+  "name": "<package_name>",  // The Webpack container name matches the NPM package name, with "@" replaced by "." and "/" removed.
+  "exposedModules": {
+    "PluginRoot": "./src/index.ts"  // The default module name is "PluginRoot" and doesn't need explicit specification in the app-config.yaml file.
+  }
+}
+----
+
+You can add a `scalprum` section to the `package.json` file. For example:
+
+.Example `scalprum` customization
+[source,json]
+----
+"scalprum": {
+  "name": "custom-package-name",
+  "exposedModules": {
+    "FooModuleName": "./src/foo.ts",
+    "BarModuleName": "./src/bar.ts"
+    // Define multiple modules here, with each exposed as a separate entry point in the Webpack container.
+  }
+}
+----
+
+Dynamic plugins might need adjustments for {product-short} needs, such as static JSX for mountpoints or dynamic routes. These changes are optional but might be incompatible with static plugins.
+
+To include static JSX, define an additional export and use it as the dynamic plugin's `importName`. For example:
+
+.Example static and dynamic plugin export
+[source,tsx]
+----
+// For a static plugin
+export const EntityTechdocsContent = () => {...}
+
+// For a dynamic plugin
+export const DynamicEntityTechdocsContent = {
+  element: EntityTechdocsContent,
+  staticJSXContent: (
+    <TechDocsAddons>
+      <ReportIssue />
+    </TechDocsAddons>
+  ),
+};
+----
+--
+
+.Prerequisites
+* The `@janus-idp/cli` package is installed. Use the latest version (`@latest` tag) for compatibility with the most recent features and fixes.
+* The third-party plugin must have a valid package.json file in its root directory, containing all required metadata and dependencies.
+* Node.js and NPM is installed and configured.
+* The third-party plugin is compatible with your {product} version. For more information, see link:https://github.com/janus-idp/backstage-showcase/blob/main/docs/dynamic-plugins/versions.md[Version compatibility matrix].
+
+.Procedure
+Use the `package export-dynamic-plugin` command from the `@janus-idp/cli` package to export the plugin:
```

Comment:
Is this a single-step procedure, if so, I'd suggest making it a single item in an unordered list - see these examples: [Sample single-step procedures](https://docs.google.com/document/d/1e46KXZaSU2rxfZu6rP2bn3710RoTJpzeD2vlh8kRDG0/edit?tab=t.0)